app/util/Transcoder.scala

```diff
@@ -0,0 +1,40 @@
+package util
+
+import javax.inject.{Inject, Singleton}
+
+import akka.actor.{ActorSystem, Scheduler}
+import akka.agent.Agent
+import com.amazonaws.services.elastictranscoder.model.ListJobsByPipelineRequest
+import model.transcoder.JobStatus
+
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+
+class Transcoder @Inject()(awsConfig: AWSConfig) {
+
+  val agent: Agent[List[JobStatus]] = Agent(List.empty)
```

Comment:
Could we just request the pipeline state on-demand, rather than caching the results?